diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/files/SemicolonsRuleFixTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.diktat.ruleset.chapter3.files
+
+import com.saveourtool.diktat.ruleset.rules.chapter3.files.SemicolonsRule
+import com.saveourtool.diktat.util.FixTestBase
+import generated.WarningNames
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class SemicolonsRuleFixTest : FixTestBase("test/paragraph3/semicolons", ::SemicolonsRule) {
+    @Test
+    @Tag(WarningNames.REDUNDANT_SEMICOLON)
+    fun `should remove redundant semicolons`() {
```

Comment:
maybe add a new rule which fails on current version and passes after the fix?

If it's used two rules -- you can add it to smoke test for example

Reply:
Done